### PR TITLE
refactored readme according release

### PR DESCRIPTION
**Pull request by Gifford47**
The old readme was a little bit unclear und a lot of informations was missing. I tried to summarize all info that i currently have. Feel free to add as much info as you have :)

### CODE REVIEW

**Comment by Gifford47** [comment]
the info from discussion #38 has to be added also

File: README.md [hunk 1]
```diff
@@ -1,48 +1,50 @@
...
+  * HC-SR04 proximity sensor (car (below) detection)
+  * HC-SR501 PIR motion sensor
+* Efficient MQTT traffic (send only MQTT Message if states changed)
 
 ## Web Interface
 
 ***http://[deviceip]***
 
-![alt text](Images/webinterface.PNG)
+![alt text](Images/webinterface.png)
```
Comment (kfroeschl) [inline README.md:39]: Link is not working - filename is case sensitive
Images/webinterface.PNG

File: README.md [hunk 2]
```diff
@@ -77,82 +80,136 @@ Response (JSON)
...
+
+<details>
+<summary>HomeAssistant shutter cards</summary>
+
+<br>
 
-![image](https://user-images.githubusercontent.com/14005124/215218504-bddf65e2-6c88-4d0a-83bd-de3cacb63c88.png)
-![alt text](Images/HA.png)
+![Homeassistant card1](Images/ha_shuttercard.png)
+![Homeassistant card2](Images/ha.png)
```
Comment (kfroeschl) [inline README.md:214]: wrong filename again
Images/HA.png

**Review by kfroeschl — CHANGES_REQUESTED** [review]
looks good and well structured 👍

Only some images do not show

**Comment by Gifford47** [comment]
could someone provide a new pic from the webinterface with the configuration tab?

**Comment by Tysonpower** [comment]
> could someone provide a new pic from the webinterface with the configuration tab?

here it is:

image

**Review by Gifford47 — COMMENTED** [review]
all fine

**Pull request merged**